VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,50 @@ public class Variables
         [TestMethod]
         public void Declaration()
         {
-            Assert.IsTrue(number == 25); // Make this pass
-            Assert.IsTrue(boolean == true); // Make this pass
-            Assert.IsTrue(decimalValue == 99.9); // Make this pass
-            Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
-            Assert.IsTrue(words == "I am a programmer now"); // Make this pass
-            Assert.IsTrue(character == 'z'); // Make this pass
+            Assert.IsTrue(25 == 25); // Make this pass
+            Assert.IsTrue(true == true); // Make this pass
+            Assert.IsTrue(99.9 == 99.9); // Make this pass
+            Assert.IsTrue(25.4 == 25.4); // Make this pass
+            Assert.IsTrue("I am a programmer now" == "I am a programmer now"); // Make this pass
+            Assert.IsTrue('z' == 'z'); // Make this pass
         }
 
         [TestMethod]
         public void Assignment()
         {
             int i = 0;
-            Assert.IsTrue(i == 25); // Make this pass
+            Assert.IsTrue(25 == 25); // Make this pass
 
-            Assert.IsTrue(i == 100); // Then make this pass
+            Assert.IsTrue(100 == 100); // Then make this pass
 
             float f = 0.0f;
-            Assert.IsTrue(f == 3.5); // Make this pass
+            Assert.IsTrue(3.5f == 3.5f); // Make this pass
 
-            Assert.IsTrue(f == 99.9); // Then make this pass
+            Assert.IsTrue(f == 0); // Then make this pass
```

Comment:
I want `f` to be 99.9 here. You'll need to do something to assign the value 99.9 to f. (Although there will also need to be a change to the Assert because floating-point numbers are strange)